docs/docs/guides/testing.md

```diff
@@ -24,3 +24,78 @@ test('Test multipliedState', () => {
   expect(testSnapshot.getLoadable(multipliedState).valueOrThrow()).toBe(100);
 })
 ```
+
+## Testing Recoil Selectors inside of a React Component
+
+It can be useful to know the state inside a component when testing it. You can use the state to evaluate the user's actions, comparing it with an expected state. As described [`here`](/docs/guides/dev-tools#observing-all-state-changes), you can use this component designed for the developer tools, also for testing your code.
```

Comment:
That section in the guide on dev tools focuses on getting a Snapshot for all state.  That can be useful, and perhaps worth another example, but your example below just uses the normal `useRecoilValue()` hook.  This is great and simple!  But, I guess keep the reference to the dev tools guide in a separate section from this example then.

Reply:
I can remove the reference from here. Maybe mixing `useRecoilValue` with the snapshot mechanics can confuse people.